administration/scheduling-and-retries.md

```diff
@@ -64,26 +76,29 @@ The following example configures the **scheduler.base** as 3 seconds and **sched
 
 The waiting time will be:
 
-| Nth retry | waiting time range (seconds) |
-| --- | --- | 
+| Nth retry | Waiting time range (seconds) |
+| --- | --- |
 | 1 | (3, 6)  |
 | 2 | (3, 12) |
 | 3 | (3, 24) |
 | 4 | (3, 30) |
 
-## Configuring Retries
+## Configure retries
 
-The Scheduler provides a simple configuration option called **Retry\_Limit**, which can be set independently on each output section. This option allows us to disable retries or impose a limit to try N times and then discard the data after reaching that limit:
+The scheduler provides a configuration option called `Retry_Limit`, which can be set
+independently on each output section. This option lets you disable retries or
+impose a limit to try N times and then discard the data after reaching that limit:
 
 |  | Value | Description |
```

Comment:
Do we need this first column if these are all for Retry_Limit?

Reply:
I honestly don't know and I decided that was outside my scope here